### PR TITLE
[BUG] return value of debuglogger expected to be an error or empty st…

### PR DESCRIPTION
**Pull request by hummal**
…ring. If no empty string debuglogger returns undefined

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -311,6 +311,7 @@ module.exports = function (options, callback) {
       return msg
     }
     debuglogger(msg)
+    return ''
```
Comment (pocketjoso) [inline src/index.js]: I see, this is an old code path that hasn't really been maintained.

Comment (pocketjoso) [reply src/index.js]: To make a little bit more sense, please return `msg` instead. Also feel free to do an `if/else` with the `console.error/debuglogger` and put the (shared) `return msg` after the `if/else`. I.e.

Comment (hummal) [reply src/index.js]: but msg is not allowed to be more than "". If I use msg, there is always content in strErr. I would still need to set msg = ''

Comment (pocketjoso) [reply src/index.js]: > but msg is not allowed to be more than ""

Comment (pocketjoso) [reply src/index.js]: Better to have it as you had it and just return `` when no error. Sorry I didn't realise exactly what you were trying to fix before.

**Pull request merged**